banzai_nres/main.py

```diff
@@ -8,41 +8,32 @@
 July 2018
 """
 
-from banzai_nres.bias import BiasMaker as nres_BiasMaker
-from banzai_nres.dark import DarkMaker as nres_DarkMaker
-
-from banzai import bias, trim, dark, gain
-from banzai import qc
+from banzai_nres import settings
 from banzai.main import process_directory, parse_directory_args
-from banzai import main as banzai_main
-from banzai.context import TelescopeCriterion
-import operator
 import logging
 
 logger = logging.getLogger(__name__)
 
-NRES_CRITERIA = [TelescopeCriterion('camera_type', operator.contains, 'NRES'),
-                 TelescopeCriterion('schedulable', operator.eq, True)]
-
-banzai_main.ORDERED_STAGES = [qc.HeaderSanity,
-                              qc.ThousandsTest,
-                              qc.SaturationTest,
-                              bias.OverscanSubtractor,
-                              gain.GainNormalizer,
-                              trim.Trimmer,
-                              bias.BiasSubtractor,
-                              dark.DarkSubtractor]
-
 
 def make_master_bias(pipeline_context=None, raw_path=None):
-    pipeline_context, raw_path = parse_directory_args(pipeline_context, raw_path, NRES_CRITERIA)
-    process_directory(pipeline_context, raw_path, ['BIAS'], last_stage=trim.Trimmer,
-                      extra_stages=[bias.BiasMasterLevelSubtractor, nres_BiasMaker],
+    pipeline_context, raw_path = parse_directory_args(pipeline_context, raw_path, settings.NRES_CRITERIA,
+                                                      settings.IMAGE_CLASS)
+    process_directory(pipeline_context, raw_path, ['BIAS'], last_stage=settings.BIAS_LAST_STAGE,
```

Comment:
You can use `settings.BIAS_IMAGE_TYPES` instead of `['BIAS']`, also for the other calibration types below